sql/group.go

```diff
@@ -125,6 +130,19 @@ func (n *groupNode) Next() bool {
 	for n.plan.Next() {
 		values := n.plan.Values()
 		for i, f := range n.funcs {
+			if f.val.Distinct {
+				encoded := make([]byte, 0, 100)
+				encoded, n.err = encodeTableKey(encoded, values[i])
```

Comment:
I imagine this doesn't work for expressions like `COUNT(DISTINCT (x, y))`. I think you'll have to special case `parser.DTuple` and loop over the values calling `encodeTableKey` for each one.